React/Base/RCTUIKit.h

```diff
@@ -479,6 +479,16 @@ NS_INLINE CGRect CGRectValue(NSValue *value)
 #define RCTUISlider UISlider
 #else
 @interface RCTUISlider : NSSlider
+
+@property (nonatomic, readonly) BOOL pressed;
+@property (nonatomic, assign) float value;
+@property (nonatomic, assign) float minimumValue;
+@property (nonatomic, assign) float maximumValue;
+@property (nonatomic, strong) NSColor *minimumTrackTintColor;
+@property (nonatomic, strong) NSColor *maximumTrackTintColor;
+
+- (void)setValue:(float)value animated:(BOOL)animated;
+
```

Comment:
Curious if you have an opinion: is it worth just copying the paper implementation for all of these views into a proper shim?

Reply:
Currently, we are just copying over the properties that are used in Fabric, however this file won't scale very well.
We could copy them to separate shim files, it would definitely make merges easier.

Reply:
#1611 filed for this, just so it's documented. Resolving this comment for now.